diff --git a/node_modules/next-auth/core/types.d.ts b/node_modules/next-auth/core/types.d.ts
index 79a4cfe..8886e2d 100644
--- a/node_modules/next-auth/core/types.d.ts
+++ b/node_modules/next-auth/core/types.d.ts
@@ -4,6 +4,7 @@ import type { TokenSetParameters } from "openid-client";
 import type { JWT, JWTOptions } from "../jwt";
 import type { LoggerInstance } from "../utils/logger";
 import type { CookieSerializeOptions } from "cookie";
+import type {CombinedSession} from 'src/types/session'
 export declare type Awaitable<T> = T | PromiseLike<T>;
 export type { LoggerInstance };
 /**
@@ -463,7 +464,7 @@ export interface DefaultSession {
  * [`SessionProvider`](https://next-auth.js.org/getting-started/client#sessionprovider) |
  * [`session` callback](https://next-auth.js.org/configuration/callbacks#jwt-callback)
  */
-export interface Session extends DefaultSession {
+export interface Session extends CombinedSession {
 }
 export declare type SessionStrategy = "jwt" | "database";
 /** [Documentation](https://next-auth.js.org/configuration/options#session) */
diff --git a/node_modules/next-auth/src/core/types.ts b/node_modules/next-auth/src/core/types.ts
index 9759211..41e5d4e 100644
--- a/node_modules/next-auth/src/core/types.ts
+++ b/node_modules/next-auth/src/core/types.ts
@@ -18,6 +18,7 @@ import type { CookieSerializeOptions } from "cookie"
 import type { NextApiRequest, NextApiResponse } from "next"
 
 import type { InternalUrl } from "../utils/parse-url"
+import {CombinedSession} from 'src/types/session';
 
 export type Awaitable<T> = T | PromiseLike<T>
 
@@ -489,7 +490,7 @@ export interface DefaultSession {
  * [`SessionProvider`](https://next-auth.js.org/getting-started/client#sessionprovider) |
  * [`session` callback](https://next-auth.js.org/configuration/callbacks#jwt-callback)
  */
-export interface Session extends DefaultSession {}
+export interface Session extends CombinedSession {}
 
 export type SessionStrategy = "jwt" | "database"
 
